airbyte-webapp/src/components/connection/CatalogTree/CatalogSection.tsx

```diff
@@ -166,7 +166,12 @@ const CatalogSectionInner: React.FC<CatalogSectionInnerProps> = ({
   );
 
   const destName = prefix + (streamNode.stream?.name ?? "");
-  const configErrors = getIn(errors, `schema.streams[${streamNode.id}].config`);
+  const configErrors = getIn(
+    errors,
+    isNewStreamsTableEnabled
+      ? `syncCatalog.streams[${streamNode.id}].config`
+      : `schema.streams[${streamNode.id}].config`
```

Comment:
I wonder why these are different. They should be the same 🤔

Reply:
As I understand correctly documentation, we used a custom format for errors.

![image](https://user-images.githubusercontent.com/66960359/210434532-916f89ae-a6b1-4410-ab97-fb626674f213.png)

In my case, I have an empty error object with this path `schema.streams[${streamNode.id}].config` due to different logic for error handling (yup way). Regarding yup schema, I have the correct error object with this path `syncCatalog.streams[${streamNode.id}].config`

![image](https://user-images.githubusercontent.com/66960359/210434783-9fbea24a-e8d0-42a6-922a-4a0909e6735f.png)